### ISSUE TITLE
Privacy Policy and Terms should be HTML (not PDF)

### ISSUE THREAD

**Issue by epicfaace** [WorldHealthOrganization/app#812]
Right now, clicking on "Privacy Policy" or "Terms" leads you to have to download a PDF and open it in a PDF reader. It would be a lot smoother if the user could just view these links in a HTML page, or maybe a modal within the app itself.

**Comment by purpledrosophila** [comment]
I can help with this, if you're open to new collaborators again. :)

**Comment by kassim** [comment]
to keep the package size down we can just link to the Google Docs PDF Viewer like so:
https://docs.google.com/viewer?url=https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/876905/300320_Revised_Guidance.pdf

**Comment by purpledrosophila** [comment]
Are the PDFs accessible for people with disabilities? If not, having them
in PDF form could be problematic, and people with disabilities are being
hit hard by the pandemic already.

**Comment by purpledrosophila** [comment]
I agree! Let me know when I can help. 🙂👍

**Comment by epicfaace** [comment]
Yes -- @purpledrosophila if you are able to modify the webserver to serve HTML files with the terms / privacy content instead of PDFs, that would be great.

**Comment by purpledrosophila** [comment]
Count me in! I will see what I can do. Should I modify any of the content,
or just change the format for now? I think, with my technical writing
experience, I could at least do an edit for clarity if needed.

**Comment by epicfaace** [comment]
Thanks! Please just change the format for now.

We can deal with the content in a separate PR; that may require input from legal as well.

**Comment by purpledrosophila** [comment]
OK. One last question then -- where are the PDFs? I can't find them in the
repository.

**Comment by epicfaace** [comment]
Here they are:
- https://whocoronavirus.org/terms
- https://whocoronavirus.org/privacy

If you could convert them to Markdown and add them to the `docs` directory (so that they are published to GitHub Pages), that would be great.

**Comment by purpledrosophila** [comment]
To my understanding, converting through Markdown won't ensure that the
resulting pages are accessible. I'll start working on this, but is there a
way I can preview the pages after conversion, then maybe make further
accessibility edits? Hope this isn't a dumb question; I haven't used GitHub
Pages before.

**Comment by purpledrosophila** [comment]
Committed an initial pull request, let me know what I can do next. :)

**Comment by avkvirtru** [comment]
I'm taking a quick look at #868 from an a11y perspective…

**Comment by purpledrosophila** [comment]
@epicfaace @avkvirtru @kassim @hspinks 
Hi all, still waiting for confirmation on exactly which files need to be converted into HTML and what those files need to look like, in terms of if I need to use a template or create a new one.

**Comment by brunobowden** [comment]
@purpledrosophila
It would be great if you could move ahead on this in the best way you can. It's incredibly helpful just to make progress and then we'll adapt if we need something different. A clean, simple design that shows all the text, links and section should be sufficient. The two pages that need conversion are the Privacy Policy and Terms of Service.

https://github.com/WorldHealthOrganization/app/blob/master/docs/legal/PRIVACY.md
https://github.com/WorldHealthOrganization/app/blob/master/docs/legal/TERMS.md

The documents should be kept as markdown but be used to automatically generate the html versions. These will replace the PDFs that are currently available here:

https://whocoronavirus.org/privacy
https://whocoronavirus.org/terms

**Comment by brunobowden** [comment]
@purpledrosophila - thanks for your previous work on #868

**Comment by purpledrosophila** [comment]
This is all done and merged. Is there something else that needs to be done
at the moment?

**Comment by epicfaace** [comment]
I think we would want people to be shown the HTML content instead of being linked to a PDF page when clicking on "Terms of Service" and "Privacy Policy" on the onboarding screen -- right @advayDev1 ?

**Comment by purpledrosophila** [comment]
I'm confused, didn't I do this by converting to Markdown for conversion
with an HTML utility in #868?

**Comment by epicfaace** [comment]
@purpledrosophila When a user clicks on "Terms of Service" in the app, it goes to https://whocoronavirus.org/terms (still the PDF version) rather than the HTML version. I've made a new issue to track the remaining work needed for this, though (#1313).
